resources/dicts/events/new_cat/general/general.json

```diff
@@ -761,5 +761,30 @@
             "tragedy_survivor1"
         ],
         "new_name": false
+    {
+        "camp": "any",
+        "tags": [
+            "m_c",
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare"
+        ],
+        "reputation": [
+            "welcoming",
+            "neutral",
+            "hostile"
+        ],
+        "event_text": "A member of o_c reveals that they have a litter which m_c is the second parent of, and that {PRONOUN/n_c/subject} now {VERB/n_c/want/wants} to join c_n, much to everycat's surprise. {PRONOUN/n_c/subject} {VERB/n_c/introduce/introduces} {PRONOUN/n_c/self} as n_c.",
```

Comment:
I'll need to get you to remove this - the event code isn't complex enough to get a litter and parent to generate with a relationship to one of the clancats. We could work on a patrol for this though - that code CAN generate a new cat with a litter of kittens that're sired by one of the members of the patrol, if you're interested

Reply:
okay! maybe i'll try patrols later on but for now i'll just get rid of it